docs/faq.md

```diff
@@ -161,13 +160,43 @@ See [the Semgrep development philosophy](/contributing/semgrep-philosophy/) for
 
 ### Comparing Semgrep to Snyk
 
-For SCA, reachability analysis is critical to cut down noise and reduce false positives. Semgrep offers reachability analysis for [several languages, such as Java, JavaScript, and Ruby](supported-languages/#semgrep-supply-chain) whereas Snyk only offers reachability for Java.
+#### SAST
+
+Both Semgrep and Snyk offer SAST solutions that are out of the box. Semgrep makes it easier to customize the rules that run against your code. Furthermore, when you know which rules were run, you can analyze your results to see if the relevant vulnerabilities were caught.
+
+In addition to selecting your rules, Semgrep allows you to write custom rules to capture use cases driven by your organization's goals. To help you write rules, Semgrep Editor provides a structure mode to guide you through the process, allows you to test your in-progress rules, and adds them to your organization’s [Policies page](/semgrep-code/policies). Semgrep offers rule-writing capabilities to all users, while Snyk limits it to Enterprise users.
+
+Both Semgrep and Snyk offer remediation advice for findings identified during scans. Snyk displays its recommendations in its UI and CLI, while Semgrep displays remediation advice and guidance in its UI and in the form of PR or MR comments.
+
+Snyk and Semgrep both display prioritization metrics, such as severity level, to help you decide which findings you should work on first. For SAST, Snyk encapsulates this information into a priority score, which provides you with information on the impact and actionability related to the finding. Semgrep, on the other hand, provides recommendations through Assistant, which offers functionality like GPT-4-powered security recommendations to help you review, triage, and remediate your Semgrep findings.
```

Comment:
The paragraph starts by talking about prioritization metrics - but we compare the Snyk score with Assistant's recommendations? The two of them are not exactly the same thing - we could focus on severity/confidence/likelihood (which are all SAST metrics separate from EPSS).